packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -456,7 +456,7 @@ class _HeroFlight {
 
   final _OnFlightEnded onFlightEnded;
 
-  Tween<Rect> heroRectTween;
+  Animatable<Rect> heroRectTween;
```

Comment:
Here and elsewhere: It's confusing that it is called "tween", but it's actually an animatable.